stylesheet/main.css

```diff
@@ -28,23 +28,37 @@ header nav a {
 }
 header nav a:hover {
   
-     text-decoration: underline;
+    border-color: red;
 }
 
 header ul {
     list-style-type: none;
     margin: 0;
     padding: 0;
     
+    display: flex;
+    justify-content: center;
 }
 
-
 main ul {
     list-style-type: circle;
 }
 
-
-
+@media screen and ( max-width: 768px ) {
+    header ul {
+        flex-direction: column;
+        text-align: center;
+        padding-top: 1em;
+        padding-bottom: 1em;
+    }
+    header nav a {
+        border-top: none;
+    }
+    header nav a:hover {
+        border-top: none;
+        text-decoration: underline;
+    }
+}

```

Comment:
You made many of these changes directly in your main css, but you need to make them in additional rules inside of media queries as you see here